### PR TITLE
Abdul/accelerometer

### PR DESCRIPTION
**Pull request by v-abdrahman**
Created a folder for Accelerometer app & added all the project files for windows 10 Iot Enterprise.

Following changes are done:

1) file modified : Accelerometer.csproj
    TargetPlatformVersion from 10.0.17763.0 to 10.0.19041.0 and TargetPlatformMinVersion from 10.0.10240.0 to 10.0.15063.0.

2) file modified : README.md
    Adding windows 10 iot Enterprise in products.
    Updated the Windows Platform from IoT Core to IoT Enterprise
    Changed the raspberry pi to Up-Board and its pins.
    Added the app *packages installing steps

### CODE REVIEW

File: Accelerometer/CS/README.md [hunk 1]
```diff
@@ -0,0 +1,298 @@
...
+* <a name="SPI_Accelerometer"></a>an [ADXL345 accelerometer board from Sparkfun](https://www.sparkfun.com/products/9836) with pin headers soldered on
+
+* a breadboard and a couple of male-to-female connector wires
+
+Visit the **UpBoard** sections below depending on which device you have:
+
+![Electrical Components](../../Resources/components.png)
+
+#### UpBoard
+If you have a UPBOARD, we need to hook up power, ground, and the SPI lines to the accelerometer.
```
Comment (neilsh-msft) [inline Accelerometer/CS/README.md]: UPBOARD is all capitals here, UpBoard or Up-Board elsewhere.  Please be consistent.  Also it shoudl be "an UpBoard" not "a UpBoard"

File: Accelerometer/CS/Accelerometer.csproj [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+    <TargetPlatformMinVersion>10.0.10240.0</TargetPlatformMinVersion>
+    <MinimumVisualStudioVersion>14</MinimumVisualStudioVersion>
+    <EnableDotNetNativeCompatibleProfile>true</EnableDotNetNativeCompatibleProfile>
+    <FileAlignment>512</FileAlignment>
+    <ProjectTypeGuids>{A5A43C5B-DE2A-4C0C-9213-0A381AF9435A};{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}</ProjectTypeGuids>
+    <PackageCertificateKeyFile>Accelerometer_TemporaryKey.pfx</PackageCertificateKeyFile>
+    <NuGetPackageImportStamp>
+    </NuGetPackageImportStamp>
+    <RuntimeIdentifiers>win10-arm;win10-arm-aot;win10-x86;win10-x86-aot;win10-x64;win10-x64-aot</RuntimeIdentifiers>
+    <GenerateAppInstallerFile>False</GenerateAppInstallerFile>
```
Comment (neilsh-msft) [inline Accelerometer/CS/Accelerometer.csproj:24]: GenerateAppInstallerFile though HoursBetweenUpdateChecks additions aren't in the pull requests for the other C# samples, what is different about this sample?

File: Accelerometer/CS/Package.appxmanifest [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+
+<Package
+  xmlns="http://schemas.microsoft.com/appx/manifest/foundation/windows10"
+  xmlns:mp="http://schemas.microsoft.com/appx/2014/phone/manifest"
+  xmlns:uap="http://schemas.microsoft.com/appx/manifest/uap/windows10"
+  IgnorableNamespaces="uap mp">
+
+  <Identity
+    Name="6b970cdc-1e41-44a0-b4c5-af6e175e6bfe"
+    Publisher="CN=Accelerometer"
```
Comment (neilsh-msft) [inline Accelerometer/CS/Package.appxmanifest]: Why was Publisher changed from "CN=MSFT" to "CN=Accelerometer"?  This should be changed back and the line above changed from Name="6b970cdc-1e41-44a0-b4c5-af6e175e6bfe" to Name="Accelerometer" to make it consistent with the other C# applications such as hello blinky.

File: Accelerometer/CS/README.md (hunk 1 above)
Comment (v-abdrahman) [reply Accelerometer/CS/README.md]: Made the changes as mentioned in the readme document.

File: Accelerometer/CS/Package.appxmanifest (hunk 3 above)
Comment (v-abdrahman) [reply Accelerometer/CS/Package.appxmanifest]: Identity name and Identity Publisher name is changed as required, and well make it consistent in all the other samples as well

File: Accelerometer/CS/README.md [hunk 4]
```diff
@@ -0,0 +1,286 @@
...
+1. Navigate to the **Advance** tab and select the **Hat Configuration** and select "LPSS SPISupport" as "Enabled".
+
+1. For more information, please review the [UP Board Firmware Settings](https://www.annabooks.com/Articles/Articles_IoT10/Windows-10-IoT-UP-Board-BIOS-RHPROXY-Rev1.3.pdf).
+  
+### Run the app
+The SPIAccelerometer app will deploy and start, and you should see accelerometer data show up on screen.
+If you have your accelerometer flat on a surface, the Z axis should read close to 1.000G, while X and Y are close to 0.000G. The values will fluctuate a little even if the device is standing still.
+This is normal and is due to minute vibrations and electrical noise. If you tilt or shake the sensor, you should see the values change in response. Note that this sample configures the device in 4G mode, so you wont be able to see G readings higher than 4Gs.
+
+![SPI Accelerometer running](../../Resources/spiaccelerometer_screenshot.png)
```
Comment (rsameser) [inline Accelerometer/CS/README.md:92]: Make sure this image appears

**Review by rsameser — CHANGES_REQUESTED** [review]
Please make sure images are appearing in the ReadMe

File: Accelerometer/CS/Accelerometer.csproj (hunk 2 above)
Comment (v-abdrahman) [reply Accelerometer/CS/Accelerometer.csproj:24]: Can you please clarify, what needs to be done?

File: Accelerometer/CS/README.md (hunk 4 above)
Comment (v-abdrahman) [reply Accelerometer/CS/README.md:92]: Updated the images path, Its coming up now.

**Pull request merged**